test/data/juraset.csv

```diff
@@ -0,0 +1,360 @@
+X,Y,Rock,Land,Cd,Cu,Pb,Co,Cr,Ni,Zn
```

Comment:
This dataset has a permissive license? We need to be careful about licensing before committing the data to the repository.

Reply:
Ok! 
Unfortunately I don't know. I selected this dataset as it seems to be interesting and has geographic coordinates, which may be used in the future for geostatistical applications.
It is described in Pierre Goovaerts "Geostatistics for Natural Resources Evaluation" book, it is avaiable on Goovaerts [website](https://sites.google.com/site/goovaertspierre/pierregoovaertswebsite/download/jura-data), and is included in the [`compositions`](https://cran.r-project.org/web/packages/compositions/index.html) and [`gstat`](https://github.com/r-spatial/gstat) R packages.
Besides the `DataDeps.jl` package, another possibility would be to use one of the datasets from `RDatasets.jl`. There are some datasets available in that package which contain compositional data, though none of them with coordinates. Datasets would have to be saved as csv files in a temporary folder as well.

Reply:
That is a really nice PR. Sorry for the delay reviewing it. I will dig deeper to see if is fine to copy the dataset with some header with the original license without GPL implications.

Reply:
I think we will be better off by downloading the dataset on the fly with DataDeps.jl according to this question: https://opensource.stackexchange.com/questions/5564/using-gpl-covered-dataset-in-non-gpl-code

Do you think it would be easy to start a fresh PR with the code changes in this one plus the download of the dataset with DataDeps.jl? If you need an example of how to do the download, here is a snippet of code that does it for another dataset:

```julia
# download dataset if needed
register(DataDep("NewZealand",
         "Taranaki Basin Curated Well Logs",
         "https://zenodo.org/record/3832955/files/taranaki-basin-curated-well-logs.tar.gz",
         "608f7aad5a4e9fded6441fd44f242382544d3f61790446175f5ede83f15f4d11",
         post_fetch_method=DataDeps.unpack))

# name of the CSV file in the dataset
csv = joinpath(datadep"NewZealand","taranaki-basin-curated-well-logs","logs.csv")
```

This snippet of code will download the dataset, and unpack it in the Julia home directory. The variable `csv` will contain the final path to the file that can be used in the tests.

Reply:
Sorry for taking a bit long to conclude the PR; time was a little scarce this past weeks.
I squashed the previous commits using git rebase. Do you think this is ok, or should I submit a fresh PR?
I added DataDeps.jl and RData.jl as dependencies, but I imagine this may not be desirable. What would be your recommendation for test specific package dependencies?